internal/server/v2/facet/golden/series_facet/series_facet.json

```diff
@@ -33,58 +33,66 @@
               ]
             },
             {
-              "facet_id": "217147238",
+              "facet_id": "1151455814",
               "observations": [
                 {
-                  "value": 5
+                  "value": 28
```

Comment:
Question for @chejennifer from @shifucun on this: https://github.com/datacommonsorg/mixer/pull/1286#discussion_r1365860210

Reply:
This is good. @chejennifer has a plan to update the meaning of this value later.